app/routes.php

```diff
@@ -455,11 +457,15 @@ function ( Application $app, Request $request ) use ( $ffFactory ) {
 	}
 );
 
-$app->get( '/', function ( Application $app ) {
-	return $app->handle(
-		Request::create( '/page/DonationForm', 'GET' ),
-		HttpKernelInterface::SUB_REQUEST
-	);
+$app->get( '/', function ( Application $app, Request $request ) use ( $ffFactory ) {
```

Comment:
The `page` route can't pass those values into the template. Also, a subrequest does not automatically pass on variables sent by `GET` or `POST`.